Sources/TuistKit/Services/Migration/MigrationCheckEmptyBuildSettingsService.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+import TSCBasic
+import TuistMigration
+import TuistSupport
+
+class MigrationCheckEmptyBuildSettingsService {
+    // MARK: - Attributes
+
+    private let emptyBuildSettingsChecker: EmptyBuildSettingsChecking
+
+    // MARK: - Init
+
+    init(emptyBuildSettingsChecker: EmptyBuildSettingsChecking = EmptyBuildSettingsChecker()) {
+        self.emptyBuildSettingsChecker = emptyBuildSettingsChecker
+    }
+
+    // MARK: - Internal
+
+    func run(xcodeprojPath: String, target: String?) throws {
+        try emptyBuildSettingsChecker.check(xcodeprojPath: AbsolutePath(xcodeprojPath), targetName: target)
```

Comment:
this seems to rely on `xcodeprojPath` being absolute, although it can be relative, too ?